src/marqo/tensor_search/tensor_search.py

```diff
@@ -293,6 +294,10 @@ def _add_documents_unstructured(config: Config, add_docs_params: AddDocsParams,
                     # Generate exactly 1 chunk with the custom vector.
                     chunks = [f"{field}::{copied[field]['content']}"]
                     embeddings = [copied[field]["vector"]]
+                    # If normalize_embeddings is true and the index version is > 2.12.0, normalize the embeddings.
+                    # We have added version specific check here to prevent backwards compatibility issues.
+                    if marqo_index.normalize_embeddings and marqo_index.parsed_marqo_version() > semver.VersionInfo.parse("2.12.0"):
```

Comment:
Does this apply to structured index as well? Also can we add a test case?

Reply:
Added same logic for structured index, also added a test case